src/transform/mod.rs

```diff
@@ -71,6 +71,29 @@ pub enum TxType {
   H_FLIPADST = 15,
 }
 
+impl TxType {
+  /// Compute transform type for inter chroma.
+  ///
+  /// <https://aomediacodec.github.io/av1-spec/#compute-transform-type-function>
+  #[inline]
+  pub fn uv_inter(self: Self, uv_tx_size: TxSize) -> Self {
+    use TxType::*;
+    if uv_tx_size.sqr_up() == TX_32X32 {
+      match self {
+        IDTX => IDTX,
+        _ => DCT_DCT,
```

Comment:
The particular set of cases that I encountered in `test_encode_decode::chroma_sampling_444_dav1d` was:
```
uv_tx_size.sqr_up() == TX_32X32 && tx_type != IDTX && tx_type != DCT_DCT
```
e.g.:
```
uv_tx_size == TX_32X32 && tx_type == DCT_ADST
```